app/elements/routing.html

```diff
@@ -95,6 +95,12 @@
 					page.createTeam();
 					return;
 				}
+				if(qs[i].indexOf('invitedby=') === 0 && qs[i + 1].indexOf('teamname=') === 0) {
+					vInvitedBy = qs[i].split('=')[1];
```

Comment:
Leaving out the `var` keyword is not an option as you are [leaking a global](http://stackoverflow.com/questions/5951228).